spec.html

```diff
@@ -13938,7 +13938,10 @@ <h1>
         1. If _prototype_ is not present, set _prototype_ to _realm_.[[Intrinsics]].[[%Function.prototype%]].
         1. Let _internalSlotsList_ be a List containing the names of all the internal slots that <emu-xref href="#sec-built-in-function-objects"></emu-xref> requires for the built-in function object that is about to be created.
         1. Append to _internalSlotsList_ the elements of _additionalInternalSlotsList_.
-        1. Let _func_ be a new built-in function object that, when called, performs the action described by _behaviour_ using the provided arguments as the values of the corresponding parameters specified by _behaviour_. The new function object has internal slots whose names are the elements of _internalSlotsList_, and an [[InitialName]] internal slot.
+        1. If _behaviour_ is described as async, then
```

Comment:
What's the verbiage for describing an async AC? "Let _c_ be a new async Abstract Closure"? Or "Async Abstract Closure" with capital A?

Reply:
I would say "async Abstract Closure", I think; the idea is what we will say "async function" or "async method" for the built-in function case (as in [proposal-iterator-helpers](https://tc39.es/proposal-iterator-helpers/#sec-asynciteratorprototype.every)).


We don't actually have any async ACs at the moment so it's not established yet.